build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```

Comment:
An older version is being used here. Use the latest version. Android Studio 2.2 as being used by us uses the version 2.2.0.